.github/workflows/ci.yml

```diff
@@ -44,16 +44,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@stable
```

Comment:
can you add a small comment above this to link to the upstream issue mentioning action-rs/toolchain is unmaintained? I understand the commit will tell it, but it's way faster